packages/airnode-validator/src/ois/ois.ts

```diff
@@ -48,14 +44,12 @@ export const reservedParameterNameSchema = z.union([z.literal('_type'), z.litera
 
 export const reservedParameterSchema = z.object({
   name: reservedParameterNameSchema,
-  // TODO: Is this correct? Shouldn't one of them always exist?
```

Comment:
This is not well documented, but they are indeed optional. If none is present the user supplied value is used. See: https://api3dao.atlassian.net/browse/AN-713